grove/pyqaoa/qaoa.py

```diff
@@ -66,68 +66,60 @@ def __init__(self, qvm, qubits, steps=1, init_betas=None,
         :param store_basis: (optional) boolean flag for storing basis states.
                             Default=False.
         """
+
+        # Randomize seed
```

Comment:
I think this comment is a bit misleading - we're setting the seed to the provided seed. From this comment I thought we were picking a random seed. 

Maybe `# Seed the random number generator.` might be better?